package.json

```diff
@@ -25,5 +25,10 @@
     "url": "https://github.com/hicsail/authentication-service/issues"
   },
   "homepage": "https://github.com/hicsail/authentication-service#readme",
-  "type": "module"
+  "type": "module",
+  "devDependencies": {
```

Comment:
It doesn't look like these dependencies should be in the top level `package.json`